apps/web/src/mystyles.scss

```diff
@@ -58,6 +58,8 @@
 
 @import "~bulmaswatch/lumen/overrides.scss";
 
+/* TODO: Plobably after PR this root elements will be removed */
+
 :root {
     --spacing-small: 8px;
     --spacing-medium: 12px;
```

Comment:
I think everything that you move to the CSS should be removed from here, that's the only way we can know the other stuff _really_ works

Reply:
ok